.github/workflows/ci-browser.yml

```
@@ -0,0 +1,46 @@
+name: ci-browser
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - packages/browser/**
+      - ./.github/workflows/ci-browser.yml
```

```suggestion
      - packages/base/**
      - packages/browser/**
      - ./.github/workflows/ci-browser.yml
```

base is a dependency of browser. Meaning that changes there could make this build fail.